### ISSUE TITLE
Autosome

### ISSUE THREAD

**Issue by ccaio** [Tree-of-Sex/toso#89]
### In which category is your new term request?

- [X] new_definition
- [ ] existing_definition

### Preferred term label

Autosome

### Synonym type

None

### Textual definition

A type of chromosome in the basic nuclear set, not including sex chromosomes or supernumerary chromosomes. 

Note that this term is mainly relevant in organisms that have both sex chromosomes and non-sex-determining chromosomes in an individual organism

### Reference

GO_0030849

### Additional context

We have added to original definition to highlight that other types of chromosome exist (apart from sex chromosomes and autosomes)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ValenzuelaLab** [comment]
Autosomes are linkage groups that form the representative karyotype of a taxon.

**Comment by KamilSJaron** [comment]
@ValenzuelaLab Is the representative karyotype the karyotype of germline? Would you say Germ-Line Restricted chromosomes are autosomes too then? And cross-connected to the definition of #90 - different stages of development would give different answers (the in springtail example - is it representative pre-elimination or post elimination).

Sorry about all those weird systems that make all the definitions harder 🙃

**Comment by anyatorgasheva** [comment]
how about - Autosome is a linkage group not involved in sex determination and present in all cells of an organism?

**Comment by KamilSJaron** [comment]
> how about - Autosome is a linkage group not involved in sex determination and present in all cells of an organism?

I think that would almost work. The only problem are silly things like red blood cells with no nuclear genome whatsoever... But if we say "Autosome is a linkage group not involved in sex determination and typically present in somatic and germ cells" it would work.

**Comment by ASEMBER2** [comment]
I agree with the definition by Kamil:
"Autosome is a linkage group not involved in sex determination and typically present in somatic and germ cells"

With reagrds to the original definition: It is hard to say what to do with B chromosomes here. B chromosomes may occasionally also be intimately involved in sex determination. Also, if we talk about Bs, then autosomes are rather called A chromosomes in this context. Sex chromosomes vs. autosomes, A chromosomes vs. B chromosomes, macrochromosomes vs. microchromosomes...

**Comment by KamilSJaron** [comment]
I think A vs B, sex vs auto, macro vs micro are all orthogonal. There can be B autosome, and B that is sex chromosome, no?

**Comment by anyatorgasheva** [comment]
I also agree with Kamil's definition "Autosome is a linkage group not involved in sex determination and typically present in somatic and germ cells"

**Comment by ValenzuelaLab** [comment]
I just re-read my comment from 21 Oct 2024 and noticed that it was incomplete!

I favor this definition:

Autosomes are linkage groups other than sex chromosomes that form the representative karyotype of a taxon.

Karyotype is defined elsewhere see Issue#8 as: The chromosomal complement in the cells of an individual, including the number of chromosomes and any abnormalities. 

Somatic cells, germ cells, and gametes may differ in their karyotypes as their chromosome numbers may vary. The representative karyotype of a taxon would be that in somatic cells. Anucleated cells such as red blood cells in mammals do not have a karyotype. 

For the definition of autosomes, I would favor using "other than sex chromosomes" instead of "not involved in sex determination" as some autosomal genes may be involved in sex determination (including autosomal modifiers of sex-determining genes from sex chromosomes and sex-determining genes in polygenic systems of GSD. A question has been whether such instances make the chromosomes carrying such genes sex chromosomes, and the ToS working groups have been reticent to call them sex chromosomes after much discussion).

For the definition of autosomes, I would also avoid "typically present in somatic and germ cell" as it is not necessary and leads to potential confusion.  

About the other points raised above:
- Autosomes can be micro or macro - no problem there.
- B-chromosomes are supernumerary dispensable chromosomes (found only in some individuals or populations, or in a subset of cells or tissues), often parasitic (selfish genetic elements) but sometimes conferring a functional advantage, can become a sex chromosome or derive from sex chromosomes. So they are not autosomes since they are not part of the representative karyotype of the taxon since they are supernumerary. For those that give rise to sex chromosomes, once they do become sex chromosomes they stop being B-chromosomes and are simply sex chromosomes. Some of the experts on B-chromosomes may correct this if mistaken.
-Germline-restricted chromosomes are autosomes eliminated fully from the soma, and present in one copy in male germline and in two copies in the female germline of songbirds. Thus, in these cases the karyotype of male and female germline varies, but not the definition of autosomes.

**Comment by astridboehne** [comment]
I find the "not involved in sex determination" bit complicated since there are system in which loci on autosomes contribute to sex determination and there is no dedicated sex chromosome (polygenic sex determination)

**Comment by lukkrat** [comment]
> I find the "not involved in sex determination" bit complicated since there are system in which loci on autosomes contribute to sex determination and there is no dedicated sex chromosome (polygenic sex determination)

I agree with Astrid, it depends on our definition of sex chromosome and sex determination. Can the definition be negative? E.g. A chromosome which is not a sex chromosome or a B chromosome.

**Comment by PaulYannJay** [comment]
Suggestion:
An autosome is a nuclear chromosome that is neither a sex chromosome, a B chromosome or a germline-restricted chromosome

**Comment by guigueny-crypto** [comment]
> Suggestion: An autosome is a chromosome that is neither a sex chromosome, a B chromosome or a germline-restricted chromosome

a nuclear

**Comment by PetrNguyen** [comment]
Autosome is a nuclear A chromosome, which is not under sex-linked inheritance, present in both soma and germline.

**Comment by Lucija27** [comment]
> Autosome is a nuclear A chromosome, which is not under sex-linked inheritance, present in both soma and germline.

I agree with this definition.

**Comment by RoelMWouters** [comment]
A chromosome = Autosomal chromosome no? So it's a loop. Suggested: an Autosome is a nuclear chromosome, which is not under sex-linked inheritance, present in both soma and germline.

**Comment by lauuragon** [comment]
> A chromosome = Autosomal chromosome no? So it's a loop. Suggested: an Autosome is a nuclear chromosome, which is not under sex-linked inheritance, present in both soma and germline.

I agree with this definition

**Comment by RoelMWouters** [comment]
But I can see how the definition: An autosome is a nuclear chromosome, which is not under sex-linked inheritance, present in both soma and germline. Doesn't exclude a B chromosome.

**Comment by PetrNguyen** [comment]
> A chromosome

In cytogenetics, A chromosomes are the “standard” or “normal” chromosomes that make up a karyotype, not autosomes.

**Comment by PetrNguyen** [comment]
> But I can see how the definition: An autosome is a nuclear chromosome, which is not under sex-linked inheritance, present in both soma and germline. Doesn't exclude a B chromosome.

That is why we use "A chromosome" as used in cytogenetics

**Comment by lukkrat** [comment]
What about "a nuclear chromosome present in the same numbers in standard karyotypes of all individuals in a population"? I do not like "standard", but aneuploids also have autosomes....

**Comment by ValenzuelaLab** [comment]
I propose the following:

Essential nuclear chromosomes other than sex chromosomes that form the representative karyotype shared by somatic and germ cells of a taxon.

This definition distinguishes autosomes as non-sex-chromosomes, and excludes B chromosomes which are not essential and GRCs that are only present in the germline.
